layers/gpu/core/gpuav_record.cpp

```diff
@@ -367,7 +375,15 @@ void Validator::PreCallRecordCmdDrawIndirect(VkCommandBuffer commandBuffer, VkBu
         return;
     }
 
-    InsertIndirectDrawValidation(*this, record_obj.location, *cb_state, buffer, offset, count, VK_NULL_HANDLE, 0, stride);
+    auto indirect_buffer_state = Get<vvl::Buffer>(buffer);
+    if (!indirect_buffer_state) {
+        InternalError(LogObjectList(commandBuffer, buffer), record_obj.location, "buffer must be a valid VkBuffer handle");
```

Comment:
```suggestion
        InternalError(commandBuffer, record_obj.location, "buffer must be a valid VkBuffer handle");
```

if we can't find it here, the `LogObjectList` isn't gonna be able to look it up neither

... I also wouldn't worry **too** much on these error, these are really just asserts and already better information then going `assert(false)`

Reply:
ah yes good point!